Symfony/CS/AbstractAlignFixer.php

```diff
@@ -86,4 +91,21 @@ protected function replacePlaceholder(Tokens $tokens, $deepestLevel)
 
         return $tmpCode;
     }
+
+    /**
+     * Cleans up extra line between variable and placeholder.
+     * Ex: `$ccc  = 1` becomes `$ccc = 1`.
+     *
+     * @param string $line
+     * @param string $placeholder
+     *
+     * @return string
+     */
+    private function cleanInnerSpaces($line, $placeholder)
```

Comment:
Ok I see. But I made `cleanInnerSpaces` method to calculate properly `$rightmostSymbol` and I indeed use it for replacement after.

Should I try trimming instead of calling this private method for code replacement?